helpers/parameter_generator.py

```diff
@@ -7,9 +7,12 @@
 from the information in LightGBM/include/LightGBM/config.h file.
 """
 from pathlib import Path
+from typing import List, Optional, Tuple
```

Comment:
```suggestion
from typing import Any, Dict, List, Optional, Tuple
```

Adding the additional imports needed based on other suggested changes in review comments.